shell/run_shell_cmd.go

```diff
@@ -72,14 +102,13 @@ func toEnvVarsList(envVarsAsMap map[string]string) []string {
 // Run the specified shell command with the specified arguments. Capture the command's stdout and return it as a
 // string.
 func RunShellCommandAndCaptureOutput(terragruntOptions *options.TerragruntOptions, command string, args ...string) (string, error) {
-	stdout := new(bytes.Buffer)
+	stdout := ioutil.Discard
```

Comment:
Why discard?

Reply:
Because we're capturing the output with the pipe now instead and don't need the buffer for that purpose.